### DISCUSSION TITLE
[Node Operator Question] <Block height is not increasing>

### DISCUSSION THREAD

**Discussion by AmardeepSambaru** [ethereum-optimism/developers#237] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I am running the optimism archive node. Two days back i updated the Binary of op-node and op-geth with latest Version.
After i updated the binary the Block height is not increasing and see the below logs.

Can anyone Help me.
Thanks you

### Protocol Description

op-geth version v1.101305.3
op-node version v1.5.0 
network: mainnet

### Node Logs

**op-node logs:**

Feb 05 10:27:02 fond-dingo op-node[3183045]: t=2024-02-05T10:27:02+0000 lvl=warn msg="Derivation process temporary error" attempts=265 err="temp: failed to fetch receipts of L1 block 0xbd8382f5be9e72fbd155da19fc5b919b6e337704a659cb3ab8c0149ae90f9815:17882072 (parent: 0xaf5c355c122ad7340e250249f845fb3d4a35410651948f6ef4e0e6b97b7d5ef4:17882071) for L1 sysCfg update: invalid receipts: log 24 of tx 0x5bb32ce62644d3aa22ef15f2d5bcb14c7c6dc51c118219315a0189015f23aaa8 has unexpected tx hash 0xf220c1329c1ae58d73454201ebb981c11c507bbefab7578e8dcecd534eafa40e"
Feb 05 10:27:03 fond-dingo op-node[3183045]: t=2024-02-05T10:27:03+0000 lvl=info msg="Received signed execution payload from p2p" id=0xe2b2843a2c9557835f84aad6859853b5d53332c0615c8e832e0b92b85c9e1167:115765023 peer=16Uiu2HAm7dgsqx3gtF9LKWgp5KzHRMLkRrxDaJi9t6B6yz8jxRtu
Feb 05 10:27:03 fond-dingo op-node[3183045]: t=2024-02-05T10:27:03+0000 lvl=info msg="Optimistically queueing unsafe L2 execution payload" id=0xe2b2843a2c9557835f84aad6859853b5d53332c0615c8e832e0b92b85c9e1167:115765023
Feb 05 10:27:05 fond-dingo op-node[3183045]: t=2024-02-05T10:27:05+0000 lvl=info msg="Received signed execution payload from p2p" id=0x2d21c8676d3e388e1aeb1ae8865d291f5c1e2a64bd9a76b9c056b599201b9118:115765024 peer=16Uiu2HAmGQeaBiD9gEV6bc89hnC37Q9WnVPzSNCK6Msd6v8Hj1xm
Feb 05 10:27:05 fond-dingo op-node[3183045]: t=2024-02-05T10:27:05+0000 lvl=info msg="Optimistically queueing unsafe L2 execution payload" id=0x2d21c8676d3e388e1aeb1ae8865d291f5c1e2a64bd9a76b9c056b599201b9118:115765024
Feb 05 10:27:07 fond-dingo op-node[3183045]: t=2024-02-05T10:27:07+0000 lvl=info msg="Received signed execution payload from p2p" id=0xb59d834b0337ef49369c66d8872bb4c7951160fb596395ddb5f3a4722a210a01:115765025 peer=16Uiu2HAmGQeaBiD9gEV6bc89hnC37Q9WnVPzSNCK6Msd6v8Hj1xm
Feb 05 10:27:07 fond-dingo op-node[3183045]: t=2024-02-05T10:27:07+0000 lvl=info msg="Optimistically queueing unsafe L2 execution payload" id=0xb59d834b0337ef49369c66d8872bb4c7951160fb596395ddb5f3a4722a210a01:115765025
Feb 05 10:27:09 fond-dingo op-node[3183045]: t=2024-02-05T10:27:09+0000 lvl=info msg="Received signed execution payload from p2p" id=0x511222c2736f572ddfe4478b57602b5fa56335a92566c3829a80a95d2d24c268:115765026 peer=16Uiu2HAm7dgsqx3gtF9LKWgp5KzHRMLkRrxDaJi9t6B6yz8jxRtu
Feb 05 10:27:09 fond-dingo op-node[3183045]: t=2024-02-05T10:27:09+0000 lvl=info msg="Optimistically queueing unsafe L2 execution payload" id=0x511222c2736f572ddfe4478b57602b5fa56335a92566c3829a80a95d2d24c268:115765026
Feb 05 10:27:11 fond-dingo op-node[3183045]: t=2024-02-05T10:27:11+0000 lvl=info msg="Received signed execution payload from p2p" id=0x6ae911ae4d2a1c265d1fb441d9ed0ae3e596c4cd260eb77c65465cabb878223c:115765027 peer=16Uiu2HAmGQeaBiD9gEV6bc89hnC37Q9WnVPzSNCK6Msd6v8Hj1xm
Feb 05 10:27:11 fond-dingo op-node[3183045]: t=2024-02-05T10:27:11+0000 lvl=info msg="Optimistically queueing unsafe L2 execution payload" id=0x6ae911ae4d2a1c265d1fb441d9ed0ae3e596c4cd260eb77c65465cabb878223c:115765027
Feb 05 10:27:12 fond-dingo op-node[3183045]: t=2024-02-05T10:27:12+0000 lvl=warn msg="Derivation process temporary error" attempts=266 err="temp: failed to fetch receipts of L1 block 0xbd8382f5be9e72fbd155da19fc5b919b6e337704a659cb3ab8c0149ae90f9815:17882072 (parent: 0xaf5c355c122ad7340e250249f845fb3d4a35410651948f6ef4e0e6b97b7d5ef4:17882071) for L1 sysCfg update: invalid receipts: log 24 of tx 0x5bb32ce62644d3aa22ef15f2d5bcb14c7c6dc51c118219315a0189015f23aaa8 has unexpected tx hash 0xf220c1329c1ae58d73454201ebb981c11c507bbefab7578e8dcecd534eafa40e"

**op-geth logs:**

Feb 06 21:29:32 fond-dingo geth[3321449]: INFO [02-06|21:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 06 22:29:32 fond-dingo geth[3321449]: INFO [02-06|22:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 06 23:29:32 fond-dingo geth[3321449]: INFO [02-06|23:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 07 00:29:32 fond-dingo geth[3321449]: INFO [02-07|00:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 07 01:29:32 fond-dingo geth[3321449]: INFO [02-07|01:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 07 02:29:32 fond-dingo geth[3321449]: INFO [02-07|02:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0
Feb 07 03:29:32 fond-dingo geth[3321449]: INFO [02-07|03:29:32.738] Regenerated local transaction journal    transactions=0 accounts=0

**Comment by sbvegan** [comment]
Hey @AmardeepSambaru, can you please try restarting your `op-node` and let me know if that helps?

**Reply by AmardeepSambaru** [reply]
I restarted the op-node so many times. still it is not working.
